libraries/api/src/middleware/logging.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Log an API request on completion, including information determined
+ * by the `remoteAuthentication` middleware, if present.
+ */
+const logRequest = ({builder, entry, monitor}) => {
+  const apiMethod = `${builder.serviceName}.${builder.apiVersion}.${entry.name}`;
```

Comment:
Isn't `serviceName` available from the logger name in the message that is created? Why duplicate here?

Reply:
Hm, true.  I did want to make it a bit more explicit since we have so many methods named `list` for example, but you're right -- this is redundant.